app/src/main/java/io/appium/uiautomator2/utils/AXWindowHelpers.java

```diff
@@ -99,20 +106,41 @@ private static AccessibilityNodeInfo[] getWindowRoots() {
         return result.toArray(new AccessibilityNodeInfo[0]);
     }
 
+    private static AccessibilityNodeInfo getTopmostWindowRootFromActivePackage() {
+        CharSequence activeRootPackageName = checkNotNull(getActiveWindowRoot().getPackageName());
+        List<AccessibilityWindowInfo> windows = getWindows();
```

Comment:
the windows intermediate variable is also not really needed

Reply:
Refactored in  624404d9209fd60a7a184755079c486488d7d1e1.